Wikipedia/Code/WMFWelcomeAnalyticsViewController.m

```diff
@@ -33,20 +34,42 @@ - (void)viewDidLoad {
         self.toggle.on                                                           = NO;
         [[BITHockeyManager sharedHockeyManager] crashManager].crashManagerStatus = BITCrashManagerStatusAlwaysAsk;
     }
+
+    [self.buttonCaretLeft setTintColor:[UIColor wmf_blueTintColor]];
+    [self.buttonCaretLeft wmf_setButtonType:WMFButtonTypeCaretLeft];
 }
 
-- (IBAction)toggleAnalytics:(id)sender {
-    if ([(UISwitch*)sender isOn]) {
+- (IBAction)toggleAnalytics:(UISwitch*)sender {
+    if ([sender isOn]) {
         [[BITHockeyManager sharedHockeyManager] crashManager].crashManagerStatus = BITCrashManagerStatusAutoSend;
         [[NSUserDefaults standardUserDefaults] wmf_setSendUsageReports:YES];
+        
+        dispatchOnMainQueueAfterDelayInSeconds(1.4, ^{
```

Comment:
It's in the mock video (dismissing the welcome screens a second or so after activating the switch), but we could instead just show the "thank you" text for a second then go back to having the button say "continue", and then the user would have to tap it to dismiss the welcome screens.